qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/html/HtmlParser.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+* Copyright (c) 2023 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.qute.parser.html;
+
+import com.redhat.qute.parser.html.scanner.HtmlScanner;
+import com.redhat.qute.parser.html.scanner.TokenType;
+import com.redhat.qute.parser.template.HtmlElement;
+
+public class HtmlParser {
```

Comment:
I think we don't need to have an html parser, but just the scanner, because the goal is to check if it exists a html<input element with a given name and you don't need to show this input in outline, or manage some completion for input (you don't need AST).

So I think the scanner is enough.